tests/phpunit/Metrics/MinMsiCheckFailedTest.php

```diff
@@ -50,7 +50,7 @@ public function test_it_can_be_created_for_min_MSI(): void
         );
     }
 
-    public function test_it_can_be_created_for_min_covered_code_MSI(): void
+    public function test_it_can_be_created_for_min_covered_code__msi(): void
```

Comment:
These changes seems to a bit wild.

Reply:
yeah not a big fan but also not really willing to fight against PHP-CS-Fixer either tbh